optimum/bettertransformer/models/__init__.py

```diff
@@ -28,6 +27,8 @@
 
 
 BETTER_TRANFORMER_LAYERS_MAPPING_DICT = {
+    # Tapas Model
+    "TapasLayer": BertLayerBetterTransformer,
```

Comment:
Can you remove `# Tapas Model` and append TapasLayer to the section just below `# Bert Family`?